packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -150,7 +150,7 @@ class Track(Resource):
     @cache(ttl_sec=5)
     def get(self, track_id):
         decoded_id = decode_with_abort(track_id, ns)
-        return get_single_track(decoded_id, None, ns)
+        return get_single_track(decoded_id, None, ns, exclude_gated=False)
```

Comment:
This feels like a small change that could have a big impact to any client flows that are using it and _not_ expecting gated tracks to come back?

Reply:
This is definitely true, and the only part of this change that makes me somewhat nervous - I need to double check where (if anywhere) we hit this endpoint in our client. I suspect it's nowhere but good to be sure.